### PR TITLE
Send weekly list with security bugs that need to be un-hidden

### LINKED ISSUE #1841: Un-hide security bugs that are marked as duplicates of already-open bugs
We have a *LOT* (804 at the time of writing) of hidden security bugs which are marked as duplicates of other bugs. A cursory glance shows many that go back years to bugs which were already un-hidden long ago. Is this something we can/should automate?

### LINKED ISSUE #1855: Do not copy metadata to bug with the `[no-nag]` tag

### PR DESCRIPTION
**Pull request by suhaibmujahid**
Fixes #1841 

## Checklist

- [ ] Type annotations added to new functions
- [ ] Docs added to functions touched in main classes
- [x] Dry-run produced the expected results
- [ ] The [`to-be-announced`](https://github.com/mozilla/relman-auto-nag/labels/to-be-announced) tag added if this is worth announcing

### CODE REVIEW

**Comment by suhaibmujahid** [comment]
By matching `core-security` group I found only 84 duplicate bugs (gets filtered to 27 bugs).

@rvandermeulen suggested matching any group that has the "sec" substring, which catches 856 duplicate bugs (gets filtered to 304 bugs).

**Comment by dveditz** [comment]
In issue 1841 I meant to do a substring match when I said "contains `core-security`", I didn't mean an exact match (I'm sorry I wasn't very precise). The bugs will have miscellaneous groups like dom-core-security, javascript-core-security, core-security-release, etc. Only a subset will have literal "core-security".

File: bugbot/rules/security_unhide_dups.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        return False
+
+    def get_summary(self, bug):
+        # This will prevent the default behavior of hiding the summary of
+        # security bugs.
+        return bug["summary"]
+
+    def handle_bug(self, bug, data):
+        bugid = str(bug["id"])
+        data[bugid] = bug
```
Comment (marco-c) [inline bugbot/rules/security_unhide_dups.py:27]: Nit: 
```suggestion
        data[str(bug["id"])] = bug
```

**Review by marco-c — APPROVED** [review]

File: bugbot/rules/security_unhide_dups.py (hunk 1 above)
Comment (suhaibmujahid) [reply bugbot/rules/security_unhide_dups.py]: Done in bb20bb358122391a93461f39122d467b32b9ff47. I used `bugid` as a kind of consistency with other rules.

**Review by marco-c — APPROVED** [review]

**Pull request merged**